webapp/tests/test_functions.py

```diff
@@ -4650,6 +4650,29 @@ def mock_evaluateTokens(reqCtx, tokens, replacements=None):
 
         self.assertEqual(result, expectedResults)
 
+    # test_minMax
+    def test_minMax(self):
```

Comment:
Not sure if we want to test function like this. Please check other tests. Usually, you need to pick 2-3-5 etc predefined series, apply your function and compare results with right one.
Function below is just comparing your function with some reference implementation - but if we know that reference implementation is fine - why do we need to check that at all?